lib/ansible/modules/cloud/azure/azure_rm_cdnendpoint.py

```diff
@@ -0,0 +1,676 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2018 Hai Cao, <t-haicao@microsoft.com>, Yunge Zhu <yungez@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: azure_rm_cdnendpoint
+version_added: "2.8"
+short_description: Manage a Azure CDN endpoint.
+description:
+    - Create, update, start, stop and delete a Azure CDN endpoint.
+
+options:
+    resource_group:
+        description:
+            - Name of a resource group where the Azure CDN endpoint exists or will be created.
+        required: true
+    name:
+        description:
+            - Name of the Azure CDN endpoint.
+        required: true
+    state:
```

Comment:
just for consistency, we always put "state" at the end

Reply:
fixed